### ISSUE TITLE
Saved Usernames not populating

### ISSUE THREAD

**Issue by doug-episource** [daishi4u/ForceCode#82]
Thanks for taking this over -- it is a fantastic extension and you've done great work adding new features. However, I cannot figure out how to use the "Saved Usernames" I have logged into several Orgs, but the side panel never populates or picks up the new Usernames.

System info
ForceCode 3.8.4
VSCode 1.27.1
MacOS 10.12.6 Sierra

**Comment by doug-episource** [comment]
Okay, I think I have figured out how to populate the names -- you need to do them manually in the force.json file, right?

However, the next issue is failed OAuth when trying to switch usernames. Where do you store the cached OAuth credentials?

**Comment by daishi4u** [comment]
The oauth stuff is stored by the salesforce cli in the .sfdx folder in your home directory. Expiration of refresh tokens depends on your orgs settings so if it's set to expire instantly then the usernames might not save

**Comment by doug-episource** [comment]
Cool, thanks for that, I found those files. One issue I am having is that clicking the Saved Username does not log me into a different Org. Instead, it prompts the login page and then OAuth fails - the page does not redirect, and hitting <enter> on the URL gives an OAuth Error.

> OAuth Error
> We can't authorize you because of an OAuth error. For more information, contact your Salesforce administrator.
> 
> OAUTH_APPROVAL_ERROR_GENERIC : An unexpected error has occured during authentication. Please try again.

I can switch by logging in, but only after closing VSCode. It doesn't seem to like logging into another Org during the same session.

Thanks for the help!

**Comment by daishi4u** [comment]
On my phone it looks like you commented on this, but maybe it got removed?

**Comment by doug-episource** [comment]
Hi Brett, yes I deleted my comment because it wasn't really what I wanted to post. I have been able to login into different Orgs, but switching sometimes does not work. I have not been successful off and on in switching by clicking on the Saved Usernames. It does not automatically login with OAuth, however, and has me redo OAuth. The problem with that is that sometimes the OAuth fails with an Error and I would need to close and reopen VSCode. Oh, the joys of OAuth.

I have made sure that have logged out of all Orgs in Chrome, because it has been my experience that it can really mess up OAuth. If I can get consistent reproduction steps, I will post here, but so far today has been pretty good.

Again, thanks for putting so much work into this extension!
